docs/how-to/configure-build-interval.md

```diff
@@ -0,0 +1,24 @@
+# How to configure `build-interval`
+
+You can configure how often the image will be built by specifying the `build-interval` configuration option.
+
+```
+BUILD_INTERVAL=3
+OPENSTACK_AUTH_URL=<openstack-auth-url, e.g. http://my-openstack-deployment/openstack-keystone>
+OPENSTACK_PASSWORD=<openstack project password>
+OPENSTACK_PROJECT_DOMAIN_NAME=<openstack project domain name>
+OPENSTACK_PROJECT_NAME=<openstack project name>
+OPENSTACK_USER_DOMAIN_NAME=<openstack user domain name>
+OPENSTACK_USERNAME=<openstack username>
+
+juju deploy github-runner-image-builder \
+--config build-interval=$BUILD_INTERVAL
```

Comment:
```suggestion
BUILD_INTERVAL_IN_HOURS=3
OPENSTACK_AUTH_URL=<openstack-auth-url, e.g. http://my-openstack-deployment/openstack-keystone>
OPENSTACK_PASSWORD=<openstack project password>
OPENSTACK_PROJECT_DOMAIN_NAME=<openstack project domain name>
OPENSTACK_PROJECT_NAME=<openstack project name>
OPENSTACK_USER_DOMAIN_NAME=<openstack user domain name>
OPENSTACK_USERNAME=<openstack username>

juju deploy github-runner-image-builder \
--config build-interval=$BUILD_INTERVAL_IN_HOURS
```
The first thought that came to mind is the unit of time (I know you explain it below, but for the impatient reader it might be better to explain it directly).